src/main/kotlin/com/sourcegraph/cody/agent/CodyAgentService.kt

```diff
@@ -107,16 +107,17 @@ class CodyAgentService(private val project: Project) : Disposable {
       }
 
       agent.client.onOpenUntitledDocument = Function { params ->
-        // We always get a file which does not start with file:// there.
-        // They start with untitled:// which is VSCode-specific way
-        // to say that files does not exist on the disk yet.
-        val vf =
-            CodyEditorUtil.createFileOrScratch(project, params.uri, params.content)
-                ?: return@Function false
+        val result = CompletableFuture<Boolean>()
```

Comment:
Can you put a brief comment here explaining what the CompletableFuture setup here is for? (If it's only needed for Windows then it would be good to know why.) Thanks!